integration-tests/helpers/index.js

```diff
@@ -271,15 +271,18 @@ async function createSandbox (dependencies = [], isGitRepo = false,
   }
   const folder = path.join(os.tmpdir(), id().toString())
   const out = path.join(folder, `dd-trace-${version}.tgz`)
-  const allDependencies = [`file:${out}`].concat(cappedDependencies)
 
   await fs.mkdir(folder)
+  await execHelper('command -v bun >/dev/null 2>&1 || npm i -g bun@1.2.23', { env: restOfEnv })
   const preferOfflineFlag = process.env.OFFLINE === '1' || process.env.OFFLINE === 'true' ? ' --prefer-offline' : ''
-  const addCommand = `yarn add ${allDependencies.join(' ')} --ignore-engines${preferOfflineFlag}`
   const addOptions = { cwd: folder, env: restOfEnv }
-  await execHelper(`npm pack --silent --pack-destination ${folder}`, { env: restOfEnv }) // TODO: cache this
+  await execHelper(`bun pm pack --silent --destination ${folder}`, { env: restOfEnv }) // TODO: cache this
```

Comment:
If we're going to switch the "packer" for integration tests, it should be whatever we use to publish.

This means either:

1. Use Bun to publish to npm, or
2. Use yarn, for this test helper, since that's what we use for package publishing (afaik? correct me if I'm wrong on that one).

The objective of integration tests is _not_ speed, but accuracy. We should be as close to customer reality as we can here, and using a different tool for packaging is a step further away from that.